src/server.rs

```diff
@@ -21,6 +21,15 @@ pub struct TlsStream<IO> {
     pub(crate) state: TlsState,
 }
 
+impl<IO> TlsStream<IO> {
+    pub fn peer_certificates(&self) -> Option<Vec<Vec<u8>>> {
```

Comment:
Approving #39 which does this already.

Reply:
There was already a pull request for returning the certificates directly ( #25 ) which was rejected in part because @skade didn't want to expose rustls. I think he has a point. A rustls certificate is just a Vec<u8> anyway.

Regarding references. I guess we could consider returning an iterator over &Vec<u8> here if you want to avoid copying the whole vector by default.

The commit adding documentation by @nacardin in #39 should be added though.